### DISCUSSION TITLE
SWR as an alternative to Redux

### DISCUSSION THREAD

**Discussion by hnsnl15** [vercel/swr#2185] [Q&A]
Hello everyone,

I'm new to SWR, can it be used as an alternative to other state management tools like redux? And if so, how can you implement it?

**Comment by dev-afzalansari** [comment]
> I'm new to SWR, can it be used as an alternative to other state management tools like redux? And if so, how can you implement it?

hi @hnsnl15, it's already answered #587

**Reply by hnsnl15** [reply]
Thanks man, that's a newbie question up there.
